src/components/core/PivotTable.tsx

```diff
@@ -63,6 +64,10 @@ import { VisualizationTypes } from '../../constants/visualizationTypes';
 import { IColumnDefOptions, IGridCellEvent, IGridHeader, IGridRow } from '../../interfaces/AGGrid';
 import * as invariant from 'invariant';
 
+import InjectedIntlProps = ReactIntl.InjectedIntlProps;
+import InjectedIntl = ReactIntl.InjectedIntl;
+import { AVAILABLE_TOTALS } from '../visualizations/table/totals/utils';
+
 import '../../../styles/scss/pivotTable.scss';
 
 export interface IPivotTableProps extends ICommonChartProps {
```

Comment:
I think "totals" type definition is missing in this interface

Reply:
done